opensearch/src/test/java/org/opensearch/sql/opensearch/executor/OpenSearchExecutionEngineTest.java

```diff
@@ -207,6 +214,32 @@ public void onFailure(Exception e) {
     assertTrue(plan.hasClosed);
   }
 
+  @Test
+  void executeRawResponseSuccessfully() {
+    OpenSearchExecutionEngine executor = new OpenSearchExecutionEngine(client, protector);
+    Settings settings = mock(Settings.class);
+    when(settings.getSettingValue(QUERY_SIZE_LIMIT)).thenReturn(100);
+    PhysicalPlan plan = new FakeOperator(new FakeOpenSearchIndexScan(mock(OpenSearchClient.class),
+        settings, "test", 10000, mock(OpenSearchExprValueFactory.class),
+        "not empty"));
+    when(protector.protect(plan)).thenReturn(plan);
+
+    AtomicReference<QueryResponse> result = new AtomicReference<>();
+    executor.execute(plan, new ResponseListener<QueryResponse>() {
+      @Override
+      public void onResponse(QueryResponse response) {
+        result.set(response);
+      }
+
+      @Override
+      public void onFailure(Exception e) {
+        fail(e);
+      }
+    });
+
+    assertNotNull(result.get());
```

Comment:
you could check that it's equal to "not empty", right?

Reply:
Changed in https://github.com/Bit-Quill/opensearch-project-sql/commit/c7208ea1f35ee9d657fb6898e800edb77e142ab3. Thanks